### PR TITLE
TXAD-1: Implement Onboarding module

### PR DESCRIPTION
**Pull request by ruchawajpe**

### CODE REVIEW

File: assets/images/error_image.png
[the remark this answers is not in the document]
Comment (ruchawajpe) [reply assets/images/error_image.png:1]: Circle avatar background image does't accept Svg format.

File: lib/blocs/selectLanguage/select_language_bloc.dart
Comment (objectorientguy) [inline lib/blocs/selectLanguage/select_language_bloc.dart:1]: comments?

File: lib/repositories/selectLanguage/select_language_repository_impl.dart
Comment (objectorientguy) [inline lib/repositories/selectLanguage/select_language_repository_impl.dart:1]: comments?

File: lib/screens/onboarding/selectLanguage/select_language_screen.dart
Comment (objectorientguy) [inline lib/screens/onboarding/selectLanguage/select_language_screen.dart:1]: comments?

File: lib/screens/onboarding/selectTimeZone/select_time_zone_screen.dart
Comment (objectorientguy) [inline lib/screens/onboarding/selectTimeZone/select_time_zone_screen.dart:1]: class name SelectTimeZoneScreen then why 
  static const routeName = 'SelectYourTimeZoneScreen'; ??

File: lib/screens/onboarding/welcome_screen.dart
Comment (objectorientguy) [inline lib/screens/onboarding/welcome_screen.dart:1]: comments?

**Review by objectorientguy — CHANGES_REQUESTED** [review]

**Review by objectorientguy — APPROVED** [review]

**Pull request merged**